src/warc2zim/content_rewriting/ds.py

```diff
@@ -256,14 +255,15 @@ def filter_by_bitrate(array, max_bitrate, mime):
 ]
 
 
-def build_domain_specific_rewriter(path: str, url_rewriter):
+def get_ds_rules(path: str):
```

Comment:
We have no return type, while this returns conditionnaly

Reply:
Return type added